src/doip_lua_script.cpp

```diff
@@ -0,0 +1,172 @@
+#include "doip_lua_script.h"
+
+/**
+ * Default Constructor if no lua file was found.
+ */
+DoipLuaScript::DoipLuaScript(){ //if there is no config for the doip server or the ecus this default configuration will be used
```

Comment:
Not sure if "DoIPLuaScript" is a good name.
Maybe "DoIPConfigurationFile" is better

Reply:
We took the naming from the "ecu_lua_script" file that creates the simulated ecus from the script files and so does the "doip_lua_script" file for the DoIPServer instance but we can change that too

Reply:
I think that the context of these things are different. The ecu_lua_script is a _simulation_ file for an ECU. I agree that the name is not perfect, but at least it is a _script_ that contains executable parts.
On the other hand the doip_lua_script is a more or less _static configuration_ of the DoIPServer and not meant to be executed like a script. So I would somehow remove the word _script_ from the class name.